web-ng/src/app/components/project-header/project-header.component.ts

```diff
@@ -14,19 +14,42 @@
  * limitations under the License.
  */
 
-import { Component, OnInit, ElementRef } from '@angular/core';
+import { Component, OnInit, ElementRef, OnDestroy } from '@angular/core';
 import { AuthService } from './../../services/auth/auth.service';
 import { UserProfilePopupComponent } from '../../components/user-profile-popup/user-profile-popup.component';
 import { MatDialog } from '@angular/material/dialog';
 import { ShareDialogComponent } from '../share-dialog/share-dialog.component';
+import { ProjectService } from '../../services/project/project.service';
+import { Observable, Subscription } from 'rxjs';
+import { Project } from '../../shared/models/project.model';
 
 @Component({
   selector: 'app-project-header',
   templateUrl: './project-header.component.html',
   styleUrls: ['./project-header.component.scss'],
 })
-export class ProjectHeaderComponent implements OnInit {
-  constructor(public auth: AuthService, private dialog: MatDialog) {}
+export class ProjectHeaderComponent implements OnInit, OnDestroy {
+  readonly activeProject$: Observable<Project>;
+  lang: string;
+  title: string;
+  projectId!: string;
+
+  subscription: Subscription = new Subscription();
+  constructor(
+    public auth: AuthService,
+    private dialog: MatDialog,
+    private projectService: ProjectService
+  ) {
+    this.lang = 'en';
+    this.title = '';
+    this.activeProject$ = this.projectService.getActiveProject$();
```

Comment:
Since `activeProject$` isn't used elsewhere in this case, can we make it a `const` local variable?